src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Utils/CBOR+AnyMap.swift

```diff
@@ -41,12 +41,15 @@ extension CBOR {
         case .date(let dateValue):
             return dateValue as Any
         case .utf8String(let stringValue):
-            return stringValue as Any
+            return stringValue.trimmingCharacters(in: .whitespacesAndNewlines) as Any
         case .array(let arrayValue):
             return arrayValue.map { $0.anyValue }
         case .map(let mapValue):
             return mapValue.anyMap as Any
-        case .null, .undefined, .tagged, .break:
+        case .null:
+            // if the value is null we will remove both the key and value from the dictionary
```

Comment:
I don't understand this change. 
Why is this done?

A null value is a valid value.
Removing the key alters the structure of the object, in the way that it is not the same object anymore.

Reply:
The tech specs were updated, now we need to remove null values from the DGC CBOR:
<img width="1151" alt="Screenshot 2021-08-04 at 10 01 05" src="https://user-images.githubusercontent.com/15270737/128144614-bcc75c86-77e0-4db7-b96d-ab23403cd7d9.png">

Reply:
the DGC from the previous preset should be valid now if we try to scan the qr code

Reply:
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/5d0c104b512eb601d22a1926e204222d8feddf6c/docs/spec/dgc-overview-client.md#decoding-a-dgc-from-cose